asteroid/losses/pit_wrapper.py

```diff
@@ -216,7 +220,7 @@ def find_best_perm(pair_wise_losses, perm_reduce=None, **kwargs):
                 :class:`torch.Tensor`: The indices of the best permutations.
         """
         n_src = pair_wise_losses.shape[-1]
-        if perm_reduce is not None or n_src <= 3:
+        if perm_reduce is not None or n_src <= 3 or linear_sum_assignment is None:
```

Comment:
And last one.

Reply:
Hm generally speaking I don't think silently changing behaviour depending on the users environment is a good idea. It may be ok in this case, I don't know.

Reply:
Maybe for scipy we can keep the deps.